src/drivers/imx/sdma.c

```diff
@@ -750,8 +753,18 @@ static int sdma_prep_desc(struct dma_chan_data *channel,
 		return -EINVAL;
 	}
 
+	/* IMX8MPRM p.7369 says watermark level is always given in bytes.
+	 * Here it is set to 128*2/8=32 for 8ch/16b.
+	 * It works, but it also works without the / 8 (which gives watermark = 256).
+	 * SAI FIFO is 128x32bit=512bytes.
+	 * Not sure what to conclude. Likely SAI fifo packing should be taken into account?
+	 */
 	watermark = (config->burst_elems * width) / 8;
 
+	tr_info(&sdma_tr, "SDMA: burst_elems %d width %d array count %d",
+		config->burst_elems, width, config->elem_array.count);
+	tr_info(&sdma_tr, "SDMA: src_width %d dest_width %d watermark %d",
+		config->src_width, config->dest_width, watermark);
```

Comment:
Please do not add unconditional tr_info prints that run every period. That will cause a lot of spam in the logs, preventing us from seeing actually important messages. At a minimum downgrade these to tr_dbg prints, though I don't really see the value of having them in the upstream code in the first place.

I suppose you can keep these prints while the pull request is still in draft mode, if they help you debug. But remember to remove them.

Reply:
These two only run on setup, at least that's what sof-logger is showing me. I made them info because dbg is not so helpful because then the spamming you mentioned really starts. I will remove them.